utils.py

```diff
@@ -268,3 +269,26 @@ def check_authorization(self, user, url_string, *args):
 
         method(self, user, url_string, *args)
     return check_authorization
+
+
+#  SEE HOW GET POWER USERS IDEA
```

Comment:
Get users of institutions trusted that has permission to analize resquests for new institutions